doc/creating_entry_types.md

```diff
@@ -190,6 +190,24 @@ const appearanceInputs = (tabView, _options) => {
 Pageflow is looking for translation keys of the form
 `pageflow.entry_types.paged.editor.entry_metadata_configuration_attributes.manual_start.{inline_help,label}`.
 
+The `browserNotSupportedView` is a Backbone view that will be displayed if the access to editor needs to be blocked due to browser compatibility issues.
```

Comment:
Text was between a sentence, so I moved it at the end of topic 'Backbone Integration.